docs/source/quickstart.md

```diff
@@ -399,29 +351,21 @@ op.plot(ss_f, label="Final orbit")
 
 Which produces this beautiful plot:
 
-```{figure} _static/hohmann.png
----
-align: center
-alt: Hohmann transfer
----   
-Plot of a Hohmann transfer.
-```
-```{eval-rst}
-Where are the planets? Computing ephemerides
---------------------------------------------
+<img align="center" width="350" alt="Plot of a Hohmann transfer" src="_static/hohmann.png">
+
+
+##### Where are the planets? Computing ephemerides
+
+versionadded: [0.14.0](https://github.com/poliastro/poliastro/tree/v0.14.0)
```

Comment:
I have no idea how this is done in MyST. Could we perhaps leave `{eval-rst}` for this case? And/or open an issue on https://github.com/executablebooks/MyST-Parser?